doc/source/configuration/ironic.rst

```diff
@@ -0,0 +1,465 @@
+  baremetal_network: "rack-net"
+  auth:
+    auth_url: "{{ lookup('env', 'OS_AUTH_URL') }}"
+    username: "{{ lookup('env', 'OS_USERNAME') }}"
+    password: "{{ lookup('env', 'OS_PASSWORD') }}"
+    project_name: "{{ lookup('env', 'OS_PROJECT_NAME') }}"
+
+With these variables defined we can now begin deploying the baremetal nodes as
+instances, to begin we invoke the deploy-baremetal-instance ansible playbook.
+
+.. code-block:: console
+
+  kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/deploy-baremetal-instance.yml
+
+This playbook will update the Kayobe network allocations with the the admin
+network addresses associated with that rack for each baremetal server, e.g.
+in the case of rack 1 this will appear in network-allocations.yml as
+
+.. code-block:: yaml
+
+  physical_rack1_admin_oc_net_ips:
+    hv003: 172.16.208.133
+    hv004: 172.16.208.134
+
+Once the network allocations have been updated, the playbook will then create a
+Neutron port configured with the address of the baremetal node admin network.
+The baremetal hypervisors will then be imaged and deployed associated with that
+Neutron port. You should ensure that all nodes are correctly associated with
+the right baremetal instance, you can do this by running a baremetal node show
+on any given hypervisor node and comparing the server uuid to the metadata on
+the Nova instance.
+
+Once the nodes are deployed, we can use Kayobe to configure them as compute
+hosts, running kayobe overcloud host configure on these nodes will ensure that
+all networking, package and various other host configurations are setup
+
+.. code-block:: console
+
+  kayobe overcloud host configure --limit baremetal-<YOUR_BAREMETAL_HARDWARE_TYPE>
+
+Following host configuration we can begin deploying OpenStack services to the
+baremetal hypervisors by invoking kayobe overcloud service deploy. Nova
+services will be deployed to the baremetal hosts.
+
+.. code-block:: console
+
+  kayobe overcloud service deploy --kolla-limit baremetal-<YOUR_BAREMETAL_HARDWARE_TYPE>
+
+Un-enrolling hypervisors
+========================
+
+To convert baremetal hypervisors into regular baremetal compute instances you
+will need to drain the hypervisor of all running compute instances, you should
+first invoke the nova-compute-disable playbook to ensure all Nova services on
+the baremetal node are disabled and compute instances will not be allocated to
+this node.
+
+.. code-block:: console
+
+  (kayobe) $ kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/nova-compute-disable.yml
+
+Now the Nova services are disabled you should also ensure any existing compute
+instances are moved elsewhere by invoking the nova-compute-drain playbook
+
+.. code-block:: console
+
+  (kayobe) $ kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/nova-compute-drain.yml
+
+Now the node has no instances allocated to it you can delete the instance using
+the OpenStack CLI and the node will be moved back to ``available`` state.
```

Comment:
Maybe just clarify here that it's the baremetal instance that you're deleting, not the VMs that you've migrated away